### DISCUSSION TITLE
How to create custom menu div inside dropdown?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#43509] [Q&A]
Hi guys, how I can archive this result? Can I use antd mobile component? How also to create multiple links inside a dropdown of the megamenu? Anyone tried it before?

![Screenshot 2023-07-12 alle 09 03 00](https://github.com/ant-design/ant-design/assets/40239902/c60205cb-d4f0-4b62-9878-2383c308333b)

**Comment by zombieJ** [comment]
Since current Menu popup do not support FlattenMode, I think you can use Menu + Popover to realize this.

**Reply by vitalijalbu** [reply]
Is there a demo i can take inspiration?

**Reply by vitalijalbu** [reply]
@zombieJ could you pls share something? thanks a lot

**Comment by avciventures** [comment]
Is there any update or example about this topic?

**Reply by vitalijalbu** [reply]
Use popover and flex, then do ul>li
